tests/test_create_models.py

```diff
@@ -0,0 +1,34 @@
+from enclave_mgmt import create_models
+from uuid import UUID
+import pandas as pd
+
+
+def test_duplicate_user_uuid():
+    """drop_duplicate_users should return an a dataframe without duplicate
+    user_uuids when the optional filter flags are applied to compile-models
+    """
+    enrollments_data = {
+        'user_uuid': [UUID("3c2ab175-f9b9-42b5-9857-9fd789914a7c"),
```

Comment:
This test will currently pass no matter what. We need to make this UUID `123e4567-e89b-12d3-a456-426655440000` so that it fails if we don't invoke `drop_duplicates()` on the join between users and enrollments.

Just as FYI: The scenario we're trying to emulate here is when a user happens to be in multiple courses. This occurs if a teacher has multiple courses or if a student ends up for whatever reason enrolled in multiple courses which IIRC has happened in a couple of cases we've spotted.

Reply:
I see now - thanks for the correction. I've updated accordingly.